.github/workflows/push.yml

```diff
@@ -49,6 +49,35 @@ jobs:
               composer install --no-progress
               composer rector
 
+  docs:
+      name: Documentation
+      runs-on: ubuntu-latest
+      steps:
+          - uses: actions/checkout@v2
+          - name: Setup PHP
+            uses: shivammathur/setup-php@v1
+            with:
+                php-version: 7.2
+                coverage: none # disable xdebug, pcov
+          - run: |
+              composer install --no-progress
+              composer check-docs
```

Comment:
These few lines are repeated everywhere. Any way to exclude it somewhere above? Like Travis has?

Reply:
Not that I am aware of

Reply:
Well, that's kinda stupid to repeat it everywhere.

Reply:
Indeed

Reply:
If that's kinda of Yaml, this might help:
https://github.com/rectorphp/rector/blob/master/.travis.yml#L122-L145

Reply:
I tried it elsewhere and it's not support yet :/ damn

https://github.community/t5/GitHub-Actions/Support-for-YAML-anchors/m-p/31845/highlight/true#M919